src/main/java/ladder/domain/line/Height.java

```diff
@@ -0,0 +1,42 @@
+package ladder.domain.line;
+
+import java.util.Objects;
+
+public class Height {
+    private static final int MIN_HEIGHT_SIZE = 1;
+    private static final String MINIMUM_SIZE_MESSAGE = "최소값은 %d 입니다.";
+
+    private final int size;
+
+    public Height(int size) {
+        validateMinimumSize(size);
+        this.size = size;
+    }
+
+    public static Height of(int size) {
+        return new Height(size);
+    }
+
+    private void validateMinimumSize(int size) {
+        if(size < MIN_HEIGHT_SIZE) {
```

Comment:
조건문 사용시 다음과 같이 if 와 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요 :)
```suggestion
        if (size < MIN_HEIGHT_SIZE) {
```

[7.4절](https://myeonguni.tistory.com/1596)
https://naver.github.io/hackday-conventions-java/#space-between-keyword-parentheses